ui/src/projects.vue

```diff
@@ -77,6 +76,7 @@ export default {
             loading: false,
 
             query: "",
+            datatype_name : {},
```

Comment:
Let's use `datatypeName` (camel case) for all new variables (or any variable that we touch..)